test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -137,6 +138,42 @@ def allow_listed(schema):
     ("dist_c10d", datetime.date(2099, 9, 17)),
 ]
 
+def has_valid_upgraders(schema, version_map):
```

Comment:
Would you elaborate on the design thinking here? Is this just to avoid developers having to add an allow list entry when also writing an upgrader? Writing an allow list entry seems pretty easy.

Reply:
In version_map.cpp, we store the old schemas of the operators that are "upgraded". In this function, we just go over that list and make sure if the schema is one of those old signatures.